src/cargo/core/workspace.rs

```diff
@@ -46,65 +46,69 @@ use pathdiff::diff_paths;
+    /// set above.
     members: Vec<PathBuf>,
+    /// Set of ids of workspace members
     member_ids: HashSet<PackageId>,
 
-    // The subset of `members` that are used by the
-    // `build`, `check`, `test`, and `bench` subcommands
-    // when no package is selected with `--package` / `-p` and `--workspace`
-    // is not used.
-    //
-    // This is set by the `default-members` config
-    // in the `[workspace]` section.
-    // When unset, this is the same as `members` for virtual workspaces
-    // (`--workspace` is implied)
-    // or only the root package for non-virtual workspaces.
+    /// The subset of `members` that are used by the
+    /// `build`, `check`, `test`, and `bench` subcommands
+    /// when no package is selected with `--package` / `-p` and `--workspace`
+    /// is not used.
+    ///
+    /// This is set by the `default-members` config
+    /// in the `[workspace]` section.
+    /// When unset, this is the same as `members` for virtual workspaces
+    /// (`--workspace` is implied)
+    /// or only the root package for non-virtual workspaces.
     default_members: Vec<PathBuf>,
 
-    // `true` if this is a temporary workspace created for the purposes of the
-    // `cargo install` or `cargo package` commands.
+    /// `true` if this is a temporary workspace created for the purposes of the
+    /// `cargo install` or `cargo package` commands.
     is_ephemeral: bool,
 
-    // `true` if this workspace should enforce optional dependencies even when
-    // not needed; false if this workspace should only enforce dependencies
-    // needed by the current configuration (such as in cargo install). In some
-    // cases `false` also results in the non-enforcement of dev-dependencies.
+    /// `true` if this workspace should enforce optional dependencies even when
+    /// not needed; false if this workspace should only enforce dependencies
+    /// needed by the current configuration (such as in cargo install). In some
+    /// cases `false` also results in the non-enforcement of dev-dependencies.
     require_optional_deps: bool,
 
-    // A cache of loaded packages for particular paths which is disjoint from
-    // `packages` up above, used in the `load` method down below.
+    /// A cache of loaded packages for particular paths which is disjoint from
+    /// `packages` up above, used in the `load` method down below.
     loaded_packages: RefCell<HashMap<PathBuf, Package>>,
 
-    // If `true`, then the resolver will ignore any existing `Cargo.lock`
-    // file. This is set for `cargo install` without `--locked`.
+    /// If `true`, then the resolver will ignore any existing `Cargo.lock`
+    /// file. This is set for `cargo install` without `--locked`.
     ignore_lock: bool,
 
     /// The resolver behavior specified with the `resolver` field.
     resolve_behavior: ResolveBehavior,
+    /// If `true`, then workspace `rust_version` would be used in `resolve`
+    /// This is set for v3 resolver.
```

Comment:
I think this might be a bit misleading since how this gets set is a little more complicated than just the resolver version. I might say something like "This is set based on the resolver version, config settings, and CLI flags.".

Reply:
Yeah, I was honestly trying to understand the logic and simplify the comment to not get too much into details. Like the updated description, let me fix it!